### ISSUE TITLE
Edit Observation: Changes Date & Time

### ISSUE THREAD

**Issue by mmulholland** [olivierfriard/BORIS#115]
When editing an observation (Ex. renaming observations, fixing an error in the independent variables, etc.), the date/time field automatically changes to the current date and time. Can this be changed to keep the original observation date/time? Or a check box added to preferences that allows us to choose if the date/time changes with edits? Thank you, Michele

**Comment by olivierfriard** [comment]
Hi,
I am not able to reproduce this problem with the current version of BORIS (v. 6.2.2)
Can you send me more details? What version are you using? What is your OS?

**Comment by mmulholland** [comment]
Ok, I've played with it more and I think the issues lies with the imported observations from the app. I am using 5.1.3 (so that's my first issue, I'll update to the current version right now) with Windows 7. 

I am using the app on a Samsung Galaxy tablet, collect the data, send the observations to the project on my PC using the project server. At that point, the dates and times match the actual date/time of the observation, then I went into the observations on my laptop (v 5.1.3, Windows 7) and edited a couple of independent variables that had been entered incorrectly, saved and then the date/time was automatically changed to the date/time I edited the observation on my laptop. Hopefully that clarifies, but I'll go update to the current version and check if it still occurs.
